tests/test_slogpy_console.py

```diff
@@ -0,0 +1,25 @@
+"""Import slogpy from slog"""
+from slogpy.slog import Slog as slog # noqa: N813 I001
+
+"""This is a setup function"""
+def setup_function():
+    """Not to create a file for output"""
+    slog.initialize(file_logging=False)
+
+"""First unit test"""
```

Comment:
These comments are unnecessary and the use of triple quote is not appropriate for a single line comment.  But, https://peps.python.org/pep-0257/ says that using triple double quotes for a single line in a docstring is common practice.